src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -246,14 +245,29 @@ def _gen_export_id(self):
                 break
         return nid
 
+    def _check_rados_notify(self, ioctx, obj):
+        sleep_time = 0
+        while sleep_time < 60:
+            try:
+                ioctx.notify(obj)
+                break
+            except Exception as e:
+                if e.errno == e.ETIMEDOUT:
+                    sleep_time += 10
+                    log.exception(f"Ganesha timed out {e}")
+                    time.sleep(sleep_time)
+                else:
+                    raise
```

Comment:
Do we really need to retry here? If ganesha dies, then when it comes back it'll pick up the latest config. At that point the notification is not really necessary anymore.

Reply:
I agree with Jeff. I don't think we need to do a retry. We should just ignore `ETIMEDOUT`.

Reply:
@batrick @jtlayton I have updated the patch. Please take another look.